### DISCUSSION TITLE
Data availability for TRUST4 run on all TCGA RNAseq samples

### DISCUSSION THREAD

**Discussion by ashishyeri** [liulab-dfci/TRUST4#172] [General]
Hi,
Thank you for creating such a wonderful resource. Is there a way to access TRUST4 TCR and BCR clonotypes data from all TCGA RNASeq samples?  I'm hoping to try some clustering algorithms on the TCR/BCR sequences on all TGCA TCR clonotypes and was wondering if the sample level clonotypes will be made public at some point of time. Thank you very much!

**Comment by mourisl** [comment]
We are concerned that releasing the TCR/BCR sequences with TCGA sample id could violate the data access policy of TCGA, so we shared TRUST4's results after de-identifying the sample: https://zenodo.org/record/6326136#.Y44UEezMIVU . Though they don't have the exact TCGA sample ID, we provide the cancer type information. Would this help?

**Comment by felon02** [comment]
thanks for generosity

**Comment by felon02** [comment]
i try to analyze the match between neoantigen and TCR, Is it possible for you to share the full "output trust_report.tsv" file of TCGA with ID masking. That would be very convinient for many downstream softwares. thank you.
